packages/debug/src/browser/debug-session.tsx

```diff
@@ -867,7 +868,11 @@ export class DebugSession implements CompositeTreeElement {
         }
         return <div className='theia-debug-session' title='Session'>
             <span className='label'>{this.label + label}</span>
-            <span className='status'>{this.state === DebugState.Stopped ? 'Paused' : 'Running'}</span>
+            <span className='status'>
+                    {this.state === DebugState.Stopped
+                        ? nls.localizeByDefault('Paused')
+                        : nls.localizeByDefault('Running')}
+                </span>
```

Comment:
The formatting here is a bit odd, we should clean up or even extract the `status` to a variable before we `return`.

Reply:
I had just added a localization, i hadn't changed formatting) But ok, I've just made a commit with fixes)